openpype/tools/loader/widgets.py

```diff
@@ -1033,6 +1033,18 @@ class FamilyListView(QtWidgets.QListView):
     def __init__(self, dbcon, family_config_cache, parent=None):
         super(FamilyListView, self).__init__(parent=parent)
 
+        from Qt import QtCore, QtWidgets
```

Comment:
Do not import from `Qt` here. If imports need to be done then import from `qtpy`.
Also, I suspect these imports should be at the top of file.